kf_lib_data_ingest/etl/configuration/target_api_config.py

```diff
@@ -362,10 +358,10 @@ def _validate_relationships(self):
         """
```

Comment:
You can get rid of this method since we're getting rid of `relationships` in `kids_first.py`.